inmap/cmd/config.go

```diff
@@ -125,6 +128,7 @@ func ReadConfigFile(filename string) (config *ConfigData, err error) {
 	config.InMAPData = os.ExpandEnv(config.InMAPData)
 	config.VariableGridData = os.ExpandEnv(config.VariableGridData)
 	config.OutputFile = os.ExpandEnv(config.OutputFile)
+	config.Logfile = os.ExpandEnv(config.Logfile)
```

Comment:
So here we can add some logic to set the log file location to be the same as the output file, except for with .log instead of .shp.